src/uwtools/resources/jsonschema/execution-serial.jsonschema

```diff
@@ -12,6 +12,9 @@
     },
     "executable": {
       "type": "string"
+    },
+    "threads": {
+      "type": "integer"
```

Comment:
The semantics might be debatable, but my position is that OpenMP is not serial, and that a `threads` value doesn't belong in a serial config. Would it work to remove this and instead change `filter-topo.jsonschema` to `"$ref": "urn:uwtools:execution" instead of `"$ref": "urn:uwtools:execution-serial"`?

Reply:
I see what you mean. It feels a little gross that there are so many MPI-related variables in the `execution` schema. This may be one of our only programs that is threaded, but not parallelized with MPI.

Reply:
Not necessarily in this PR, but we could rename `execution.jsonschema` -> `execution-parallel.jsonschema` and `execution-serial.jsonschema` -> `execution.jsonschema` if that feels better. Or have `-parallel` and `-serial` in both names to be more explicit.

Reply:
It might be nice to have `execution-parallel` and `execution-serial`.